.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint:prettier
```

Comment:
Adding precommit formatting check

Reply:
@leahjlou I think running this at every commit for all developers can become annoying. Imagine a developer commits several times intermediate work, we don't want to run this as it's not running fast and will slow down as the number of files increases. We should configure it at the editor level or the developer can still add this as a personal git hook

Reply:
Sorry @beguene I forgot to address this comment before I merged this PR. I agree with your point, opened a new PR here to remove it https://github.com/hirosystems/explorer/pull/859

Reply:
No issue @leahjlou I actually made my comment after the merge. Thanks for fixing it.